hadoop-ozone/interface-storage/src/main/java/org/apache/hadoop/ozone/om/ratis/TransactionInfo.java

```diff
@@ -52,11 +52,23 @@ private OMTransactionInfo(String transactionInfo) {
     transactionIndex = Long.parseLong(tInfo[1]);
   }
 
-  private OMTransactionInfo(long currentTerm, long transactionIndex) {
+  private TransactionInfo(long currentTerm, long transactionIndex) {
     this.term = currentTerm;
     this.transactionIndex = transactionIndex;
   }
 
+  public boolean isInitialized() {
+    return transactionIndex == -1 && term == 0;
+  }
+
+  public int compareTo(TransactionInfo info) {
+    if (info.getTerm() == this.getTerm()) {
+      return this.getTransactionIndex() <= info.getTransactionIndex() ? -1 : 1;
+    } else {
+      return this.getTerm() < info.getTerm() ? -1 : 1;
+    }
```

Comment:
I am not sure if I am missing something but couldn't find the usage for this function.
And shouldn't compareTo() return 0 if the TransactionInfo's are same?